search-resources-api/src/test/java/no/unit/nva/search/exception/ExceptionsTest.java

```diff
@@ -6,17 +6,26 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
-public class InputExceptionTest {
+public class ExceptionsTest {
```

Comment:
Test the exceptions from the handler using the Test utils library

Reply:
Exception tested like the others